app/lib/frontend/handlers/experimental.dart

```diff
@@ -51,7 +51,7 @@ class ExperimentalFlags {
   bool get showAdminUIForAutomatedPublishing => _enabled.contains('publishing');
 
   /// Whether to show package screenshots in search listings.
-  bool get showScreenshots => _enabled.contains('screenshots');
+  bool get showScreenshots => true;
```

Comment:
Perhaps also remove 'screenshots' from the public flags?

Reply:
Was planning to do that in another PR :-)